web/src/components/common/HelpText.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+interface HelpTextProps {
+  helpText?: string;
+  defaultValue?: string;
+  error?: string;
+}
+
+const HelpText: React.FC<HelpTextProps> = ({ helpText, defaultValue, error }) => {
+  if ((!helpText && !defaultValue) || error) return null;
+  
+  // Build the combined text with markdown formatting
+  let combinedText = helpText || '';
+  if (defaultValue) {
+    const defaultText = `(Default: \`${defaultValue}\`)`;
+    combinedText = helpText ? `${helpText} ${defaultText}` : defaultText;
+  }
+  
+  return (
+    <div className="mt-1 text-sm text-gray-500">
+      <ReactMarkdown
+        components={{
```

Comment:
why not use the same <a /> alias as Label component?

Reply:
yeah, i'll make these consistent